php/7.1/Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN chmod +x /usr/local/bin/composer
 COPY ./files/entrypoint.sh /usr/local/bin
 RUN chmod a+x /usr/local/bin/entrypoint.sh
 
-# Define "docker" as current user
+# Change working directory to use the default user (ie 'docker') home
```

Comment:
It is not really the default user, we add one, but the config may differs as it is configured outside of the dockerfile

Reply:
Thanks for the fix @jmleroux . Anael is right,  maybe just `to use the home of "docker" user` is enough.
Could you also make the change on the other PHP versions of the `akeneo/php` image, please?

Reply:
maybe it's even better to remove this comment

Reply:
Totally agree @docteurklein . The command is explicit on its own. Another example of the maintenance debt of a comment :smile: